Tests/IntegrationTests/IntegrationTests.csproj

```diff
@@ -141,7 +141,9 @@
     </ProjectReference>
   </ItemGroup>
   <ItemGroup>
-    <None Include="App.config" />
+    <None Include="..\..\RoslynPluginGenerator\App.config">
+      <Link>App.config</Link>
+    </None>
```

Comment:
Check that the old app.config files have been deleted; they don't appear to be removed in this commit.